docs/rules/no-duplicates.md

```diff
@@ -61,6 +61,25 @@ import SomeDefaultClass from './mod?minify'
 import * from './mod.js?minify'
 ```
 
+### combineTypeImports
+
+By default, this rule ignores flow & typescript type imports. However you can use the option `combineTypeImports` to combine type imports with regular imports.
```

Comment:
can you explain why this is a good thing, or how it would apply to TS? In TS, it's preferred to use `import type` for all type imports, and `import` only for value imports.

Does Flow not have a separate `import type`?

Reply:
> can you explain why this is a good thing, or how it would apply to TS? In TS, it's preferred to use `import type` for all type imports, and `import` only for value imports.

TypeScript allows to not use `import type` for type imports. `typescript-eslint` has a rule to specify the syntax: [typescript-eslint/consistent-type-imports](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/consistent-type-imports.md). As you are saying it's not the preferred way, so it shouldn't be the default, but it is something users might want to set.

The main reason we would like to combine type and regular imports in our flow code is to make it easy for developers to see what the dependencies on a certain file. Some of our files have both regular and type exports so when they are imported, it's confusing to not see them combined:

## Before
```
// @flow
import { type I18nContext } from 'app/packages/i18n/useI18n';
// Other imports
import useI18n, { I18nContextProvider } from 'app/packages/i18n/useI18n';
```

## After
```
// @flow
import useI18n, { I18nContextProvider, type I18nContext } from 'app/packages/i18n/useI18n';
```

> Does Flow not have a separate import type?

Flow supports both `import type Foo` and `import { type Foo }` syntax.

Reply:
I can’t imagine why you’d prefer the combined form over the separate import vs import type statements, but at least i understand what the option would do.

Since they can’t be combined in TS, this option would only ever be useful for Flow. I’m not sure supporting only flow is worth the complexity at this time due to the current state of Flow usage in the ecosystem.

Reply:
@ljharb understood, closing this PR for now.

Reply:
Looks like in TS 4.5+, they can be combined. I'll reopen.